Description: Add remark to options.
Author: Jörg Frings-Fürst <debian@jff-webhosting.net>
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=418630
Forwarded: 
Last-Update: 2015-01-18
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: experimental/doc/scanimage.man
===================================================================
--- experimental.orig/doc/scanimage.man
+++ experimental/doc/scanimage.man
@@ -74,6 +74,9 @@ To print all available options:
   scanimage \-h
 
 .SH OPTIONS
+\fBRemark:\fR Parameter are defined by the backends. So are \-\-mode Gray and \-\-mode Grayscale in use.
+Please read the backend documentation first.
+
 Parameters are separated by a blank from single-character options (e.g.
 \-d epson) and by a "=" from multi-character options (e.g. \-\-device\-name=epson).
 
